Examples/Tests/ion_stopping/inputs_3d

```diff
@@ -0,0 +1,96 @@
+my_constants.max_ion_energy = 100.  # eV
+my_constants.max_ion_velocity12 = sqrt(2.*max_ion_energy*q_e/m_p)/clight
+my_constants.max_ion_velocity34 = sqrt(2.*max_ion_energy*q_e/(4.*m_p))/clight
+
+max_step = 10
+
+amr.n_cell = 8 8 8
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo = -1. -1. -1.
+geometry.prob_hi = +1. +1. +1.
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+boundary.particle_lo = periodic periodic periodic
+boundary.particle_hi = periodic periodic periodic
+algo.particle_shape = 1
+
+particles.species_names = ions1 ions2 ions3 ions4
+
+ions1.species_type = proton
+ions1.injection_style = "MultipleParticles"
+ions1.multiple_particles_pos_x = 0. 0. 0. 0.
+ions1.multiple_particles_pos_y = 0. 0. 0. 0.
+ions1.multiple_particles_pos_z = 0. 0. 0. 0.
+ions1.multiple_particles_vel_x = max_ion_velocity12 0. 0. max_ion_velocity12/2.
+ions1.multiple_particles_vel_y = 0. max_ion_velocity12 0. max_ion_velocity12/2.
+ions1.multiple_particles_vel_z = 0. 0. -max_ion_velocity12 -max_ion_velocity12/2.
+ions1.multiple_particles_weight = 1. 1. 1. 1.
```

Comment:
Would it make sense to add the flags:
```
ions1.do_not_deposit = 1
```
since the deposited current here should not have any influence. That could also make the test slightly faster.

Reply:
(Same remark for the other ions species.)

Reply:
Good idea. I'll add it.